wd.html

```diff
@@ -444,6 +444,276 @@ <h3>Get Title</h3>
+driver.switchTo().window(originalWindow);</code>
+ <code class=python>#Close the tab or window
+driver.close()
+
+#Switch back to the old tab or window
+driver.switch_to.window(original_window)</code>
+ <code class=cs>//Close the tab or window
+driver.Close();
+
+//Switch back to the old tab or window
+driver.SwitchTo().Window(originalWindow);</code>
+ <code class=ruby>#Close the tab or window
+driver.close
+
+#Switch back to the old tab or window
+driver.switch_to.window original_window</code>
+ <code class=javascript>//Close the tab or window
+await driver.close();
+
+//Switch back to the old tab or window
+await driver.switchTo().window(originalWindow);
+ </code>
+</pre>
+<p>Forgetting to switch back to another window handle after closing a
+ window will leave WebDriver executing on the now closed page, and will
+ trigger a <strong>No Such Window Exception</strong>.</p>
+<p>If you didn't store
+ your previous window handle, you can alternatively switch back to the
+ first handle using:</p>
+<pre>
+ <code class=java>driver.switchTo().window(driver.getWindowHandles().iterator().next());</code>
+ <code class=python>driver.switch_to.window(driver.window_handles[0])</code>
+ <code class=cs>driver.SwitchTo().Window(driver.WindowHandles[0]);</code>
+ <code class=ruby>driver.switch_to.window driver.window_handles[0]</code>
+ <code class=javascript>const allWindows= await driver.getAllWindowHandles();
+await driver.switchTo().window(allWindows[0]);</code>
+</pre>
+
+<h3>Quiting the browser at the end of a session</h3>
+<p>When you are finished with the browser session you should call quit,
+ instead of close:
+<pre>
+ <code class=java>driver.quit();</code>
+ <code class=python>driver.quit()</code>
+ <code class=cs>driver.Quit();</code>
+ <code class=ruby>driver.quit</code>
+ <code class=javascript>await driver.quit();</code>
+</pre>
+<p>Quit will:
+ <ul>
+  <li>Close all the windows and tabs associated with that WebDriver
+   session</li>
+  <li>The browser process</li>
+  <li>The background driver process</li>
+  <li>Notify Selenium Grid that the browser is no longer in use so it can
+   be used by another session (if you are using Selenium Grid).</li>
+ </ul>
+<p>Failure to call quit will leave extra background processes and ports
+ running on your machine which could cause you problems later.</p>
+<p>Some test frameworks offer methods and annotations which you can hook
+ into to tear down at the end of a test.</p>
+<pre>
+ <code class=java>/**
+ * Example using JUnit
+ * http://junit.sourceforge.net/javadoc/org/junit/AfterClass.html
+ */
+@AfterClass
+public static void tearDown() {
+    driver.quit();
+}</code>
+ <code class=cs>/*
+    Example using Visual Studio's UnitTesting
+    https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.testtools.unittesting.aspx
+*/
+[TestCleanup]
+public void TearDown()
+{
+    driver.Quit();
+}</code>
+</pre>
+<p>If not running WebDriver in a test context, you may consider using
```

Comment:
I imagine there's similar stuff in other languages, but in Java most will be using JUnit, which has a {at}AfterAll ({at}After in Junit4) in which you can put your driver.quit - this will guarantee it will get called at the end of the test and means you dont need an ugly try/catch wrapping the whole thing. Might be worth putting that in here? Maybe as an after comment or adjoining section?

Reply:
The try / catch here is to caterer for folks using Selenium to automate admin tasks with few outside libraries.  But AfterAll might be a better approach here for the JUnit example.